src/pages/changePasswordPage/ResetPasswordPage.jsx

```diff
@@ -25,137 +29,177 @@ const ResetPasswordPage = () => {
     newPassword: false,
     confirmPassword: false,
   });
+
+  useEffect(() => {
+    const searchParams = new URLSearchParams(location.search);
+    const token = searchParams.get("token");
+    axios.get(`${API_END_POINT}/api/accounts/password/token_verification/?token=${token}`).then((res)=>{
+      if(res.data.message){
+        setVerificationToken(token);
+      }
+    }).catch((error)=>{
+      if(error.response && error.response.data.status === 400){
+        notification.error({
+          message: "Error",
+          description: `${error.response.data.errors}`,
+        })
+        navigate("/forgot/password")
```

Comment:
Redirect to login page, instead of forgot/password

Reply:
I have changed

Reply:
I have changed